app/src/main/res/layout/fragment_settings.xml

```diff
@@ -102,6 +102,15 @@
                 app:title="@string/settingsExternalContentTitle"
                 tools:subtitle="@string/settingsOptionAskMe" />
 
+            <com.infomaniak.mail.ui.main.settings.ItemSettingView
+                android:id="@+id/settingsAutomaticAdvance"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:itemAction="chevron"
+                app:title="@string/settingsAutoAdvanceTitle"
+                tools:subtitle="@string/settingsAutoAdvanceDescription" />
```

Comment:
I don't think this the tools subtitle is correct, it should show an example of a selected value instead